src/main/java/com/ironz/unsafe/UnsafeAndroid.java

```diff
@@ -467,6 +522,14 @@ public void putDouble(Object obj, long offset, double newValue) {
      * @since 1.8
      */
     @TargetApi(24)
+    public void copyMemory(Object srcObj, long srcAddr, Object dstObj, long dstAddr, long bytes) {
```

Comment:
Can't find this method in last sources https://android.googlesource.com/platform/libcore/+/android-o-preview-1/ojluni/src/main/java/sun/misc/Unsafe.java